build-ps/version.sh

```diff
@@ -3,7 +3,14 @@ DIR=$(dirname $(realpath $0))
 BRANCH=$(git branch --show-current)
 IFS='.' read -r MAJOR MINOR PATCH <<< $(echo $BRANCH | awk -F'-' '{print $2}')
 EXTRA=$(echo $BRANCH | awk -F'-' '{print $3}')
-source ${DIR}/../VERSION
+if [ -f VERSION ]; then
```

Comment:
Shouldn't we check ${DIR}/../VERSION instead of VERSION ?

Reply:
fixed